undertow/src/main/java/org/wildfly/extension/undertow/HostAdd.java

```diff
@@ -69,6 +70,7 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final boolean isDefaultHost = defaultServerName.equals(serverName) && name.equals(defaultHostName);
         final int defaultResponseCode = HostDefinition.DEFAULT_RESPONSE_CODE.resolveModelAttribute(context, model).asInt();
         final boolean enableConsoleRedirect = !HostDefinition.DISABLE_CONSOLE_REDIRECT.resolveModelAttribute(context, model).asBoolean();
+        DefaultDeploymentMappingProvider.instance().addMapping(defaultWebModule, serverName, name);
```

Comment:
If more than 1 host has the same defaultWebModule, then the last one processed is going to win.

This is more likely to happen in the case of multiple cases of undefined default-web-module attribute, as ROOT.war is the default value.

TBH, I don't know if this is a problem or not, but it seems a bit odd.